### ISSUE TITLE
`zig init` does not work as the documentation says it should

### ISSUE THREAD

**Issue by l3r8yJ** [ziglang/zig#18765]
### Zig Version

0.11.0

### Steps to Reproduce and Observed Behavior

As [documentation said](https://ziglang.org/learn/getting-started/#run-hello-world) i'm able to do `zig init` in directory, but suddenly it's not working

install with brew on macOS:
1.  `brew install zig`
2. `mkdir my-project`
3. `cd my-project`
4. `zig init`

an error:

```bash
➜  cpy zig init
info: Usage: zig [command] [options]

Commands:

  build            Build project from build.zig
  init-exe         Initialize a `zig build` application in the cwd
  init-lib         Initialize a `zig build` library in the cwd

  ast-check        Look for simple compile errors in any set of files
  build-exe        Create executable from source or object files
  build-lib        Create library from source or object files
  build-obj        Create object from source or object files
  fmt              Reformat Zig source into canonical form
  run              Create executable and run immediately
  test             Create and run a test build
  translate-c      Convert C code to Zig code

  ar               Use Zig as a drop-in archiver
  cc               Use Zig as a drop-in C compiler
  c++              Use Zig as a drop-in C++ compiler
  dlltool          Use Zig as a drop-in dlltool.exe
  lib              Use Zig as a drop-in lib.exe
  ranlib           Use Zig as a drop-in ranlib
  objcopy          Use Zig as a drop-in objcopy

  env              Print lib path, std path, cache directory, and version
  help             Print this help and exit
  libc             Display native libc paths file or validate one
  targets          List available compilation targets
  version          Print version number and exit
  zen              Print Zen of Zig and exit

General Options:

  -h, --help       Print command-specific usage

error: unknown command: init
```

### Expected Behavior

Project should be initialized with `build.zig` and etc.

**Comment by aribambang** [comment]
You can initialize the project with this command if you are using version 0.11.0:

```
zig init-exe
```

`zig init` is available starting from version 0.12.0.

<img width="383" alt="image" src="https://github.com/ziglang/zig/assets/10102342/5f576e55-fe68-48c3-939c-744d062850f4">

**Comment by l3r8yJ** [comment]
@aribambang thanks, `zig init-exe` worked!
